index.js

```diff
@@ -19,6 +19,20 @@ const WORKERS = process.env.WEB_CONCURRENCY || 1;
  */
 function start() { // eslint-disable-line max-statements
   const conf = require('./config');
+
+  if (conf.traceAPIKey) {
+    /*
```

Comment:
add something to readme about the Trace addon, link to it, how to configure it, etc.